core/client/app/components/gh-tab-pane.js

```diff
@@ -17,13 +17,13 @@ var TabPane = Ember.Component.extend({
     active: Ember.computed.alias('tab.active'),
 
     // Register with the tabs manager
-    registerWithTabs: function () {
+    registerWithTabs: Ember.on('didInsertElement', function () {
         this.get('tabsManager').registerTabPane(this);
-    }.on('didInsertElement'),
+    }),
 
-    unregisterWithTabs: function () {
+    unregisterWithTabs: Ember.on('willDestroyElement', function () {
         this.get('tabsManager').unregisterTabPane(this);
-    }.on('willDestroyElement')
+    })
```

Comment:
Same with these--just use the `didInsertElement` and `willDestroyElement` hooks.